src/lib/y2partitioner/actions/controllers/encryption.rb

```diff
@@ -83,6 +91,16 @@ def actions
             end
         end
 
+        def methods
+          encrypt_methods = [Y2Storage::EncryptionMethod::LUKS1]
+          encrypt_methods << Y2Storage::EncryptionMethod::RANDOM_SWAP if swap?
+          encrypt_methods
+        end
+
+        def method
+          @method || Y2Storage::EncryptionMethod::LUKS1
```

Comment:
Constructor already assigns a default value when there is no encryption. I would simply define `#method` as an `attr_reader`.

Reply:
But, if for any reason the controller ends receiving 

```ruby
controller.method = nil
````

then it will lost the _default_ value. 

What do you think?

Reply:
With current implementation:

```
controller.method = nil
controller.method.nil? #=> false

controller.method = :foo
controller.method #=> :foo
```

I think this is wrong. If we really want to avoid nil values, I would re-define the setter as:

```
def method=(value)
  raise Exception unless value.is_a?(EncryptionMethod)
end
``` 

For now, I would simply use `attr_accessor :method`. If the user assigns something strange, then the value will not selected, but right now we are the only ones who assign such value.